bentoml/__init__.py

```diff
@@ -24,7 +24,17 @@
     # on MacOS.
     import multiprocessing
 
-    multiprocessing.set_start_method('fork')
+    try:
+        multiprocessing.set_start_method("fork")
+    except RuntimeError:
+        import logging
```

Comment:
Instead of catching a runtime error, could we first check if `multiprocessing.get_start_method() != "fork"`, only then do we set  `multiprocessing.set_start_method("fork")`?

Reply:
@ssheng As you said, it seems better to check the error condition first. Thank you for the review.